Created stock_emails.md

gueroJeff opened the pull request:
Migrated content of https://wiki.mozilla.org/L10n:Stock_Responses to GitHub docs and updated to reflect current l10n procedures and tools.

gueroJeff committed:
Created stock_emails.md

Migrated content of https://wiki.mozilla.org/L10n:Stock_Responses to GitHub docs and updated to reflect current l10n procedures and tools.

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+Here you'll find a stockpile of templated responses to inquiries we frequently receive about Mozilla L10n. They've been centralized here so that you can simply copy and paste them into emails. While there is value to having these stock responses for common inquiries, we strongly feel that there is equal value to making them as personalized has possible. Take time to adjust the template in your em ...
```
typos: as possible, personalized

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+
+[name]
+
+###Follow up to general inquiry
```
Why the switch to 3rd level title?

gueroJeff replied:
Uhh, no good reason :-\

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+Here's how you can get started:
+
+* Create an account on [Pontoon](https://pontoon.mozilla.org/teams).
+* Check our list of existing [L10n teams](https://pontoon.mozilla.org/teams) to see if an effort for your locale already exists. If an effort exists, contact the team manager to join the team and help drive the effort. If it doesn't exist, introduce your new L10n team on the [new locales newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales).
```
nit: https for the Google groups link. Same later in the doc.

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+
+* Create an account on [Pontoon](https://pontoon.mozilla.org/teams).
+* Check our list of existing [L10n teams](https://pontoon.mozilla.org/teams) to see if an effort for your locale already exists. If an effort exists, contact the team manager to join the team and help drive the effort. If it doesn't exist, introduce your new L10n team on the [new locales newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales).
+* Introduce yourself to the L10n community on the [L10n newsgroup](http://groups.google.com/group/mozilla.dev.l10n.new-locales) by subscribing and posting an introduction. In your introduction, please add some brief information about yourself e.g., professional background, interests, etc.) and what you hope to work on (e.g., "I saw that there was xx-XX localization team and I'd like to help them  ...
```
Did you want to use just dev-l10n here? You have a missing parenthesis before the first e.g.

gueroJeff replied:
Are you asking if we want to stop using dev-l10n-new?

flodolo replied:
For existing locales I think it makes sense to use dev-l10n. Also, based on the text of the link, I assumed the link the dev-l10n-new-locales was a copy and paste error from the previous paragraph

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+#Having problems with Pontoon/Pootle
+Sorry to hear that you're having problems with your L10n tool. Please do the following to allow us to troubleshoot your problem(s):
+
+* Search bugzilla to see if a bug has already been filed for your problem(s).
```
nit: Bugzilla

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+* If you can't find a discussion thread about your problem, begin a discussion topic in those newsgroups asking for help. Include as much detail as possible.
+* If no one in the discussion group can offer a solution and there are no bugs currently filed, file a new bug providing as many details about the problem as possible.
+
+If the first three steps don't solve your problem and you have to file a bug, rest assured that the Mozilla l10n-drivers will respond promptly.
```
We should probably tell them in which component to file the bug? Do we file bugs for Pootle?

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+#Having problems with hg repositories
+Sorry to hear that you're having problems with hg. Please do the following to allow us to troubleshoot your problem(s):
+
+* Search bugzilla to see if a bug has already been filed for your problem(s).
```
nit: Bugzilla

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,111 @@
+* If you can't find a discussion thread about your problem, begin a discussion topic in those newsgroups asking for help. Include as much detail as possible.
+* If no one in the discussion group can offer a solution and there are no bugs currently filed, file a new bug providing as many details about the  problem as possible.
+
+If  the first three steps don't solve your problem and you have to file a bug, rest assured that the L10n drivers will respond promptly.
```
extra blank spaces after if. Same question about where to file the bug.

gueroJeff committed:
Fixed all the thingz

We can either take this PR or close it and take #94. They're the same. Sorry for causing problems.

gueroJeff committed:
Curly quotes

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,100 @@
+Could you also please provide me with some additional information about yourself, such as languages you speak, country you live in, and technical background?
+
+You can also visit the following wiki pages to learn the details of how we localize:
+* [https://wiki.mozilla.org/L10n:Localization_Process L10n:Localization Process:] A good description of the whole process.
```
The link is broken here. What's the URL you want to point to? Since they're email templates, I would just put the plain links all over the document, or the HTML version `<a href="URL">link text</a>".

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,100 @@
+
+We’re very excited to have you join the Mozilla localization (l10n) effort! Your skills and dedication will be very valuable to your l10n team.
+
+Next step, follow the steps outline on https://wiki.mozilla.org/L10n:Contribute. When you contact your l10n team, please CC me on that email so that I can help to make sure you receive a prompt response.
```
outline -> outlined?

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,100 @@
+Here’s how you can get started:
+
+* Check our list of existing [https://pontoon.mozilla.org/teams localization communities] to see if an effort for your language already exists. If an effort exists, contact the community to join the team and help drive the effort. If it doesn’t exist, introduce your new l10n community on the [https://groups.google.com/group/mozilla.dev.l10n Mozilla localization mailing list].
+* Introduce yourself to the L10n community on the [https://groups.google.com/group/mozilla.dev.l10n Mozilla localization mailing list] by subscribing and posting an introduction. In your introduction, please add some brief information about yourself e.g., professional background, interests, etc.) and what you hope to work on (e.g., "I saw that there was a xx-XX community and I’d like to help them ...
```
The first closing parenthesis is missing an opening.

flodolo commented on misc/stock_emails.md:
```diff
@@ -0,0 +1,100 @@
+
+If the first three steps don’t solve your problem and you have to file a bug, rest assured that the l10n-drivers will respond promptly.
+
+## Having problems with hg
```
hg -> Mercurial (hg)

flodolo reviewed (commented):
Looks good, there's a question on how we want to write URLs through the templates.

gueroJeff committed:
Fixed hyperlinks and handful of typos
